### PR TITLE
(feat) Handle obs group questions in visits table

### PR DESCRIPTION
**Pull request by denniskigen**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] My work includes tests or is validated by existing tests.

## Summary

Provides a better approach for handling obs group questions from an encounter. Specifically, this PR extracts the individual group members from the parent obsGroup collection and renders their associated labels and answers as individual entries in the observations list.

## Screenshots

> Before
 
Screenshot 2023-03-22 at 1 29 20 PM

> After

Screenshot 2023-03-22 at 1 29 40 PM

## Issue

https://issues.openmrs.org/browse/O3-1773

### CODE REVIEW

**Comment by denniskigen** [comment]
The next thing that should happen (possibly as part of this PR), is that the `display` property from the parent obs gets displayed as the title of the nested obs section. That would mean observations from the lab encounter displayed in the screenshot above would have `Serum electrolytes` and `Liver function tests` as their headings.

File: packages/esm-patient-chart-app/src/visit/visits-widget/past-visits-components/visits-table/visits-table.scss [hunk 1]
```diff
@@ -88,3 +86,11 @@
   @include type.type-style('body-compact-01');
   color: $text-02;
 }
+
+.layer {
+  height: 100%;
+
+  :global(.cds--btn--primary) {
```
Comment (denniskigen) [inline packages/esm-patient-chart-app/src/visit/visits-widget/past-visits-components/visits-table/visits-table.scss:93]: This fixes the issue where the `OverflowMenu` button has a green background.

File: packages/esm-patient-chart-app/src/visit/visits-widget/visit.resource.tsx [hunk 2]
```diff
@@ -3,13 +3,7 @@ import { openmrsFetch, OpenmrsResource, Visit } from '@openmrs/esm-framework';
...
 export function useVisits(patientUuid: string) {
   const customRepresentation =
-    'custom:(uuid,encounters:(uuid,diagnoses:(uuid,display,rank,diagnosis),form:(uuid,display),encounterDatetime,' +
-    'orders:full,' +
-    'obs:(uuid,concept:(uuid,display,conceptClass:(uuid,display)),' +
-    'display,groupMembers:(uuid,concept:(uuid,display),' +
-    'value:(uuid,display)),value),encounterType:(uuid,display),' +
-    'encounterProviders:(uuid,display,encounterRole:(uuid,display),' +
-    'provider:(uuid,person:(uuid,display)))),visitType:(uuid,name,display),startDatetime,stopDatetime,patient';
+    'custom:(uuid,encounters:(uuid,diagnoses:(uuid,display,rank,diagnosis),form:(uuid,display),encounterDatetime,orders:full,obs:full,encounterType:(uuid,display),encounterProviders:(uuid,display,encounterRole:(uuid,display),provider:(uuid,person:(uuid,display)))),visitType:(uuid,name,display),startDatetime,stopDatetime,patient';
```
Comment (denniskigen) [inline packages/esm-patient-chart-app/src/visit/visits-widget/visit.resource.tsx:6]: We're now grabbing the full `obs` object from the response.

**Review by donaldkibet — APPROVED** [review]
Great! 🚀

**Pull request merged**